src/expressions.md

```diff
@@ -264,8 +264,8 @@ exist in `core::ops` and `core::cmp` with the same names.
 a few specific cases:
 
 * Before an expression used as a [statement].
-* Elements of [array expressions], [tuple expressions], [call expressions],
-  tuple-style [struct] and [enum variant] expressions.
+* Elements of [array expressions], [tuple expressions], and [call expressions]
```

Comment:
```suggestion
* Elements of [array expressions], [tuple expressions], [call expressions],
```

Reply:
For both of these, I waffled on writing it the way I suggested and the way you suggested. I think that either read perfectly okay and appreciate your feedback! Thanks!!